mediadevices.go

```diff
@@ -214,7 +215,24 @@ func selectBestDriver(filter driver.FilterFn, constraints MediaTrackConstraints)
 	}
 
 	if bestDriver == nil {
-		return nil, MediaTrackConstraints{}, errNotFound
+		var foundProperties []string
+		for _, props := range driverProperties {
+			for _, p := range props {
+				foundProperties = append(foundProperties, fmt.Sprint(&p))
+			}
+		}
+
+		err := fmt.Errorf(`
+%w:
```

Comment:
This will add empty line before the error description.
```suggestion
		err := fmt.Errorf(`%w:
```

Reply:
Ouch.., I'll fix it